iosApp/iosApp/iOSApp.swift

```diff
@@ -45,6 +46,28 @@ struct iOSApp: App {
         appDependencies.registerTaskHandlers()
         deepLinkFlow = appDependencies.initializeDeeplink()
 
+
+        let launchArguments = ProcessInfo.processInfo.arguments
+
+        if launchArguments.contains("--presetDatabase") {
+            // Enable logging
+            DatabaseHelper.Companion().initialize(dependency: appDependencies.dependencies)
+
+            Task { [self] in
+                try await self.initDatabase()
+            }
+
+        }
+
+    }
+
+    func initDatabase() async {
+        do {
+            try await DatabaseHelper.companion.clear()
+            try await DatabaseHelper.Companion().setup()
```

Comment:
Why the 2 different `companion`?

Reply:
its the same one. just accessed it differently.